pkgs/by-name/nu/nuget/package.nix

```
@@ -45,6 +45,6 @@ stdenv.mkDerivation (attrs: {
     license = licenses.mit;
     sourceProvenance = [ sourceTypes.binaryBytecode ];
     maintainers = [ maintainers.mdarocha ];
-    platforms = [ "x86_64-linux" ];
+    platforms = [ "x86_64-linux" "aarch64-linux" "x86_64-darwin" "aarch64-darwin" ];
```

At this stage I'll just do

```suggestion
    inherit (mono.meta) platforms;
```

---

good idea